ydb/library/yql/dq/actors/task_runner/task_runner_actor_local.cpp

```diff
@@ -70,13 +72,13 @@ class TLocalTaskRunnerActor
             }
         } catch (const NKikimr::TMemoryLimitExceededException& e) {
             Send(
-                ev->Sender,
```

Comment:
Whats the reason of changing that?

Reply:
I faced a problem while implementing the WakeUp callback: event was sent to the wrong worker. Decided to use ParentId here instead of a ev->Sender to prevent such mistakes in the future